wgsl/index.bs

```diff
@@ -519,7 +521,36 @@ The <dfn noexport>transpose</dfn> of an |c|-column |r|-row matrix |A| is the |r|
 The transpose of a column vector is defined by interpreting the column vector as a 1-row matrix.
 Similarly, the transpose of a row vector is defined by interpreting the row vector as a 1-column matrix.
 
-# Shader Lifecycle # {#shader-lifecycle}
+# WGSL Module # {#wgsl-module}
+
+A WGSL program is composed of a single WGSL module.
+
+A module is a sequence of optional [=directives=] followed by [=module scope=] [=declarations=] and [[#const-assert-statement|const_assert statements]].
```

Comment:
For future:  It's an odd fit to have a "statement" at module-scope, because statements are defined as specifying execution behaviour.  I've logged a followup to fix this by rephrasing const_assert as a different kind of thing.